app/src/main/res/layout/fragment_about.xml

```diff
@@ -447,11 +439,6 @@
                 android:textStyle="bold" />
 
 
-
-
-
```

Comment:
Again, why change this file at all? Possible conflict with #215

Reply:
Didn't know that it will be changed after final merge